string pose_type
float64 head_pan
float64 head_tilt
float64 lift
float64 arm
float64 grip
float64 wrist